packages/electron-commons/test-kit/assert-utils.ts

```diff
@@ -0,0 +1,20 @@
+import { expect } from 'chai';
+
+/**
+ * Asserts that value is deep equal to one of the specified expected values.
+ * @param actual value to test
+ * @param list the list of expected values
+ */
+export function oneOfDeepEqual(actual: any, list: any[]) {
+    expect(actual).to.satisfy(() => {
+        for (const expected of list) {
+            try {
+                expect(actual).to.deep.equal(expected);
```

Comment:
this function would throw for any expectation that isn't equal
also, the error message will not be descriptive

Reply:
removed this function